bundles/org.openhab.binding.goecharger/README.md

```diff
@@ -55,6 +59,7 @@ demo.items
 
 ```
 Number:ElectricCurrent     GoEChargerMaxCurrent                 "Maximum current"                       {channel="goecharger:goe:garage:maxCurrent"}
+Number:ElectricCurrent     GoEChargerMaxCurrTemp                "Maximum current temporary"                       {channel="goecharger:goe:garage:maxCurrentTemp"}
```

Comment:
Rename a bit, this sounds like current temperature, for example `MaxCurrentTemporary`